Sources/Plasma/PubUtilLib/plResMgr/plResManager.cpp

```diff
@@ -126,16 +126,21 @@ bool plResManager::IInit()
 
     kResMgrLog(1, ILog(1, "Initializing resManager..."));
 
-    if (plResMgrSettings::Get().GetLoadPagesOnInit())
-    {
+    if (plResMgrSettings::Get().GetLoadPagesOnInit()) {
         // We want to go through all the data files in our data path and add new
         // plRegistryPageNodes to the regTree for each
         std::vector<plFileName> prpFiles = plFileSystem::ListDir(fDataPath, "*.prp");
-        for (auto iter = prpFiles.begin(); iter != prpFiles.end(); ++iter)
-        {
+        for (auto iter = prpFiles.begin(); iter != prpFiles.end(); ++iter) {
             plRegistryPageNode* node = new plRegistryPageNode(*iter);
-            plPageInfo pi = node->GetPageInfo();
-            fAllPages[pi.GetLocation()] = node;
+            const plPageInfo& pi = node->GetPageInfo();
+
+            // If a page is already added with this location, add both the already known page
+            // and the newly discovered page to a set of conflicts.
+            auto pageResult = fAllPages.try_emplace(pi.GetLocation(), node);
```

Comment:
`try_emplace` is C++17

Reply:
I just noticed our standard is C++14 :cry: